### DISCUSSION TITLE
Is there a way/example we could use Prisma with JWT service like AWS Cognito?

### DISCUSSION THREAD

**Discussion by arismarko** [prisma/prisma#7529] [Q&A]
Hi all,

Just one question if you know there is a repo as an example where we could use Prisma with AWS Cognito service. 

I was told we could use https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-integrate-with-cognito.html 

But if there any more info or someone that did something similar, it would be great to share experience and thoughts.

Many thanks,
Aris

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
